### PR TITLE
More yeet

### PR DESCRIPTION
**Pull request by ikekilinc**

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 import java.util.Random;
 
+/**
+  * Big yeet.
```
Comment (ikekilinc) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: This code is crap

Comment (zteoh) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: nice comment!

Comment (ikekilinc) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: Yeah dude totally

**Comment by zteoh** [comment]
good stuff!

**Pull request merged**